content/en/apps/examples/direct-to-patient.md

```diff
@@ -0,0 +1,137 @@
+
+## More scenarios where direct-to-patient messaging can be used to support patient care
+
+### 1. Facility appointment reminders workflow
+Appointment reminders can be configured on CHT so that household members and patients can receive facility appointment reminders. Facility appointment reminders are configured as per the recommended health guidelines for specific use cases. The clinic visit appointment reminders can be configured to automatically stop once a patient completes the expected clinic visits.
+
+### 2. Active case finding by messaging household members with survey questions
+Active case finding by messaging households with survey questions about the health of family members.
+
+## Frequently Asked Questions
+##### 1. Is the patient charged for receiving or sending the SMS?
+It is possible for health programs to acquire a zero rated short code service which is free for patients and household members to send and receive texts from the short code.
+
+##### 2. What are ways to handle exceptions?
+Exceptions can occur when programs and flows do not work as expected. This may occur due to various technical issues and may be unavoidable. One possible way to handle these unwanted issues and errors is to set up the mail group such that a notification email is sent whenever there is any exception. Another method is to keep track of whether the flow was completed i.e., all the responses from start to finish nodes were received. For the participants whose flow may have been interrupted or incomplete, one may run the workflow again.
+
+##### 3. Ways to improve response rate from patients or household members?
+The workflows in RapidPro can be set up in such a way that they expire after a certain time period. Workflows can expire when there is no response from a patient within a time period. You can handle such situations by repeating the workflow i.e., sending them again to the patient after a certain time interval. Alternatively, it can be beneficial to do some research on which time to send the message. For example, a person may be busy at work and/or away from the phone during working hours. It may be easier for them to respond during early morning or later in the evening than in the afternoon
+
+<div class="row">
+{{< figure src="logoaurum.png"  link="" alt="" title="" class="col-3">}}
+{{< figure src="logouw.png"  link="" alt="" title="" class="col-3">}}
+{{< figure src="logomedic.png"  link="" alt="" title="" class="col-3">}}
+</div>
+
+## Resources
+
+VMMC code can be accessed using the following link [D2C messaging app for post-op care](https://github.com/medic/cht-post-ops-app)
+##### Training materials
+- User manual
+- Trainer of trainers slide deck
+- Flip Books (updated to include WhatsApp)
+
+##### Illustrative [Videos](https://drive.google.com/drive/folders/1AKsjdhJD3-3nXAqKTzakGAyZS9itnsk2?usp=sharing)
```

Comment:
These videos are not publicly accessible. Either upload them to the Medic youtube channel or remove this section altogether.